plugin/gcp/pubsubTopic.go

```diff
@@ -141,14 +141,11 @@ func (t *pubsubTopic) Stream(ctx context.Context) (io.ReadCloser, error) {
 	return t.newPubsubTopicWatcher(ctx)
 }
 
-func (t *pubsubTopic) Write(ctx context.Context, _ int64, b []byte) (int, error) {
+func (t *pubsubTopic) Write(ctx context.Context, b []byte) error {
```

Comment:
Similar Q. to https://github.com/puppetlabs/wash/pull/664#discussion_r358519024 regarding result.Get

Reply:
Yes.